git_pile/genbranch.py

```diff
@@ -299,6 +312,7 @@ def init(self):
             "-q", "--quiet", help="Quiet mode - do not print list of patches", action="store_true", default=False
         )
         self.parser.add_argument("-e", "--external-pile", help="Use external pile dir as input", default=None)
+        self.parser.add_argument("--rev", help="Use pile revision passed as argument")
```

Comment:
--pile-rev would be clearer IMO.

Also, we should probably be explicit about this arg conflicting with --external-pile

Reply:
Done and force-pushed.

> Also, we should probably be explicit about this arg conflicting with --external-pile

Did that by raising an error if both options are passed. Not sure we need to add that information to the help output...